core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts

```diff
@@ -218,9 +218,11 @@ popup(state){
       const yamlLint = require('yaml-lint');
       yamlLint.lint(yaml).then(() => {
         this.isvalid = true;
+        this.invalid = false;
```

Comment:
Why do we have 2 flags? isvalid and invalid?

Reply:
Both valid and invalid condition was enable at the same time so I used two flag.